examples/ts-react-chat/src/routes/queueing.tsx

```
@@ -0,0 +1,288 @@
+import { useMemo, useState } from 'react'
+import { createFileRoute } from '@tanstack/react-router'
+import { Send, Square, X, Zap } from 'lucide-react'
+import { fetchServerSentEvents, useChat } from '@tanstack/ai-react'
+import type { QueueConfig, QueuedMessage, UIMessage } from '@tanstack/ai-react'
+import { DEFAULT_MODEL_OPTION, MODEL_OPTIONS } from '@/lib/model-selection'
+import type { ModelOption } from '@/lib/model-selection'
```

ESLint

[error] `@/lib/model-selection` type import should occur before import of `@/lib/model-selection`

(import/order)